.circleci/config.yml

```diff
@@ -0,0 +1,125 @@
+defaults: &defaults
+  working_directory: ~/SalesforceMobileSDK-ReactNative
+  macos:
+    xcode: "9.4.1"
+  shell: /bin/bash --login -eo pipefail
+  environment:
+    BASH_ENV: ~/.bashrc
+    FASTLANE_SKIP_UPDATE_CHECK: true
+    CHRUBY_VER: 2.5.1
+
+version: 2
+jobs:
+  test-ios:
+    <<: *defaults
+    steps:
+      - checkout
```

Comment:
What's under the `checkout` step here (should it be `checkout:`)?

Reply:
Nothing is required under checkout (see https://circleci.com/docs/2.0/configuration-reference/#checkout).